lib/plugins/aws/package/compile/events/stream.js

```diff
@@ -32,6 +32,10 @@ class AwsCompileStreamEvents {
             maximumRecordAgeInSeconds: {
               anyOf: [{ const: -1 }, { type: 'integer', minimum: 60, maximum: 604800 }],
             },
+            functionResponseType: {
```

Comment:
Let's expect a string here instead of an array

Reply:
noted

Reply:
It will definitely be useful, but I think we can adjust it at that point - I've made a small mistake with my suggestion to change the type to string - following other similar examples, I think we should go with such definition:
```
functionResponseType: { enum: ['ReportBatchItemFailures'] }
```